gourmet/tests/test_epicurious_plugin.py

```diff
@@ -0,0 +1,77 @@
+import os.path
+import unittest
+import urllib.request
+
+from bs4 import BeautifulSoup
+
+from gourmet.plugins.import_export.website_import_plugins import epicurious_plugin
+
+
+class DummyImporter(object):
+
+    class WebParser(object):
+        pass
+
+
+class TestAllRecipesPlugin(unittest.TestCase):
+
+    url = "https://www.epicurious.com/recipes/food/views/kiwi-lime-sorbet-233181"
+
+    def _read_html(self, download=True):
+        if download:
+            with urllib.request.urlopen(self.url) as response:
+                self.text = response.read().decode("utf8")
+            return
+
+        filename = os.path.join(os.path.dirname(__file__),
+                                "recipe_files",
+                                "epicurious_com.html")
+        with open(filename, encoding="utf8") as f:
+            data = f.read()
+        return data
```

Comment:
Same issue: inconsistent return types. Please convert to `staticmethod` and always return the data